frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/review-tab/storage-review.tsx

```diff
@@ -74,40 +85,63 @@ const StorageReviewFirehose: React.FC<StorageReviewFirehoseProps> = ({
     (sc) => getAnnotations(sc, {})[DEFAULT_SC_ANNOTATION] === 'true',
   );
 
+  const rows = combinedDisks.map((combinedDisk) => {
+    return [
+      combinedDisk.getName(),
+      combinedDisk.getSourceValue(),
+      combinedDisk.getReadableSize(),
+      combinedDisk.getDiskInterface(),
+      combinedDisk.getStorageClassName(),
+      combinedDisk.getAccessModes()?.join(', '),
+      combinedDisk.getVolumeMode()?.toString(),
+    ];
+  });
+
   return (
-    <ReviewList
-      title="Storage"
-      className={className}
-      items={combinedDisks.map((combinedDisk) => {
-        return {
-          id: combinedDisk.id,
-          value: combinedDisk.toString(),
-        };
-      })}
-    >
-      {hasStorageWithoutStorageClass && (
-        <Alert
-          title={'Some disks do not have a storage class defined'}
-          isInline
-          variant={AlertVariant.warning}
-          className="kubevirt-create-vm-modal__review-tab-storage-class-alert"
-        >
-          {defaultStorageClass ? (
-            <DefaultSCUsed defaultSCName={getName(defaultStorageClass)} />
-          ) : (
-            <NoDefaultSC />
+    <>
+      {showStorages && (
+        <>
+          {hasStorageWithoutStorageClass && (
+            <Alert
+              title={'Some disks do not have a storage class defined'}
+              isInline
+              variant={AlertVariant.warning}
+              className="kubevirt-create-vm-modal__review-tab-storage-class-alert"
+            >
+              {defaultStorageClass ? (
+                <DefaultSCUsed defaultSCName={getName(defaultStorageClass)} />
+              ) : (
+                <NoDefaultSC />
+              )}
+            </Alert>
           )}
-        </Alert>
+          <Table
+            aria-label="Storage Devices"
+            variant={TableVariant.compact}
+            cells={headers}
+            rows={rows}
+            gridBreakPoint="grid-xl"
+          >
+            <TableHeader />
+            <TableBody />
+          </Table>
+        </>
+      )}
+      {!showStorages && (
+        <EmptyState variant={EmptyStateVariant.full}>
```

Comment:
```suggestion
        <EmptyState variant={EmptyStateVariant.small}>
```

should be the same as networks

Reply:
Done. Empty state was removed entirely.